### ISSUE TITLE
SSR artifacts

### ISSUE THREAD

**Issue by Zireael07** [godotengine/godot#18395]
**Godot version:**
3.0.2

**OS/device including version:**
Radeon 6630M
Windows 7 x64

**Issue description:**
I get weird artifacts when SSR is on. No amount of fiddling with SSR setting helps.

https://webmshare.com/0BXw1

(The artifacts are: horizontal lines on car window with chase cam and the flashes of ... something... on the apex with interior cam)

**Steps to reproduce:**

**Minimal reproduction project:**
Probably difficult to repro in a minimal project, as I'm guessing it's related to amount of geometry in scene.

**Comment by LinuxUserGD** [comment]
#14837 ?

**Comment by Zireael07** [comment]
The artifacts are not black - I think they are the sky color, blueish at any rate. And I run 3d, not 3d without effects.

**Comment by Zireael07** [comment]
It's been months and I saw similar artifacts in 3.1 alpha 2, but this time, the inspector is a bit nicer and I was able to figure out the cause. It's not transparent meshes, unlike what I originally thought. It's the roughness checkbox in SSR in combination with the label I have above the car.

Unticking the roughness checkbox was the only thing that helped - no amount of fiddling with other parameters and/or material roughness itself helped.

I am not closing the issue since it seems to point at a problem with the roughness parameter of the SSR.
